benchmarking/utils/__init__.py

```diff
@@ -10,28 +10,10 @@
 # on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
-from benchmarking.utils.checkpoint import (  # noqa: F401
-    add_checkpointing_to_argparse,
-    resume_from_checkpointed_model,
-    checkpoint_model_at_rung_level,
-    pytorch_load_save_functions,
-)
-from benchmarking.utils.parse_bool import parse_bool  # noqa: F401
 from benchmarking.utils.get_cost_model import (  # noqa: F401
     get_cost_model_for_batch_size,
 )
-from benchmarking.utils.config_as_json import (  # noqa: F401
-    add_config_json_to_argparse,
-    load_config_json,
-)
 
 __all__ = [
-    "add_checkpointing_to_argparse",
-    "resume_from_checkpointed_model",
-    "checkpoint_model_at_rung_level",
-    "pytorch_load_save_functions",
-    "parse_bool",
     "get_cost_model_for_batch_size",
```

Comment:
Quick code search shows that the only code referencing `get_cost_model_for_batch_size` is commented out. Is it specific to benchmarking, or is it also a general utility? Wondering if it makes sense to also move or leave as-is.

Reply:
Yeah, that is why I left it here. I implemented cost models a while ago, for a certain project. This is useful code, but right now it is not used.